README.md

```diff
@@ -103,6 +103,7 @@ via Composer:
 | [Strava](https://github.com/Edwin-Luijten/oauth2-strava)               | composer require edwin-luijten/oauth2-strava        |
 | [Stripe](https://github.com/adam-paterson/oauth2-stripe)               | composer require adam-paterson/oauth2-stripe        |
 | [Twitch](https://github.com/tpavlek/oauth2-twitch)                     | composer require depotwarehouse/oauth2-twitch       |
+| [Twitch (Helix)](https://github.com/vertisan/oauth2-twitch-helix)      | composer require vertisan/oauth2-twitch-helix       |
```

Comment:
This is auto-generated. We probably add a new `TwitchHelixProviderConfigurator`: https://github.com/knpuniversity/oauth2-client-bundle/blob/master/src/DependencyInjection/Providers/TwitchProviderConfigurator.php

And mark the old one as deprecated (including changing its name in its configurator to `Twitch (deprecated)`.